app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -180,20 +180,20 @@ def parse_openshift_image(openshift_image)
         new_result[:tag] ||= json["tag"] if json.keys.include?("tag")
 
         docker_metadata = openshift_image[:dockerImageMetadata]
+        docker_labels = parse_identifying_attributes(docker_metadata[:Config][:Labels], 'docker_labels', "openshift")
+        labels = parse_labels(openshift_image)
         new_result.merge!(
           :architecture          => docker_metadata[:Architecture],
           :author                => docker_metadata[:Author],
           :command               => docker_metadata[:Config][:Cmd],
           :entrypoint            => docker_metadata[:Config][:Entrypoint],
           :docker_version        => docker_metadata[:DockerVersion],
-          :exposed_ports         => parse_exposed_ports(
-            docker_metadata[:Config][:ExposedPorts]),
-          :environment_variables => parse_env_variables(
-            docker_metadata[:Config][:Env]),
+          :exposed_ports         => parse_exposed_ports(docker_metadata[:Config][:ExposedPorts]),
+          :environment_variables => parse_env_variables(docker_metadata[:Config][:Env]),
           :size                  => docker_metadata[:Size],
           :labels                => parse_labels(openshift_image),
-          :docker_labels         => parse_identifying_attributes(docker_metadata[:Config][:Labels],
-                                                                 'docker_labels', "openshift"),
+          :docker_labels         => docker_labels,
+          :tags                  => map_labels('ContainerImage', labels + docker_labels)
```

Comment:
Ooh, docker labels, shiny :)

[First, are these concatenable?  Is `docker_labels` array of `{:name, :value}` hashes?]
I *think* the mapping code would map both.  With same name and 2 values, you might get 2 tags.
Is that what we want?  The Right Thing is probably a new mapping column, letting user define independent mappings for k8s labels vs docker labels (and maybe open it to other kinds of custom attrs).  Though I'm not excited about making auto-tagging more complicated :-\

Are you also adding docker labels as new virtual custom-attr columns in reports?
There, would you merge a k8s label & docker label of same name into one "column"?
We probably should be consistent between direct label querying / mapped tag querying.

Are you covering docker labels only on images?
https://docs.docker.com/engine/userguide/labels-custom-metadata/ lists several entities.
P.S. This PR does way too much, but you knew that :)